### ISSUE TITLE
[1.12] Config `ADDITIONAL_CHESTS` not working

### ISSUE THREAD

**Issue by Krutoy242** [noobanidus/Lootr#189]
Blacklisting inventories not working.

My config:
```less
    S:ADDITIONAL_CHESTS <
        minecraft:dropper
        minecraft:dispenser
        iceandfire:desert_myrmex_cocoon
        iceandfire:jungle_myrmex_cocoon
        randomthings:specialchest
     >

    B:CONVERT_ALL_LOOTABLES_EXCEPT_BELOW=true
```

[lootr.cfg](https://github.com/noobanidus/Lootr/files/11460470/lootr.txt)

And this what i see (there should be Dispensers instead of Lootr chests)
![image](https://github.com/noobanidus/Lootr/assets/936618/fe17a8dd-08cf-44b8-9245-af3671bfb80c)

This issue related to https://github.com/noobanidus/Lootr/issues/183, but since its other config option, i made new isuue.

**Comment by Krutoy242** [comment]
Just found that when generating, some Pirate Ships generated fine and some not 
![image](https://github.com/noobanidus/Lootr/assets/936618/8f048452-b384-424e-8d1d-7b259becc793)

I found that its also depends on a chunk. Sometimes same ship converted in one chunk, but didnt in other 
![image](https://github.com/noobanidus/Lootr/assets/936618/5b247ec7-feae-4b71-acf5-5b6b814dd2e0)

**Comment by embeddedt** [comment]
Is this a fresh world or the same world from before?

**Comment by Krutoy242** [comment]
Fresh one. This chunks wasn't generated before.

**Comment by noobanidus** [comment]
Is this still an issue?

**Comment by Krutoy242** [comment]
Yes. Players of `E2E-E` still have some chests and dispensers converted and some not.

**Comment by noobanidus** [comment]
I'm honestly not sure about this issue then, I'm afraid. I'll leave this issue open for now; @embeddedt did you have any thoughts on this now?

**Comment by noobanidus** [comment]
Nobody is working on 1.12 any more, sorry.
